stories/Message.story.tsx

```diff
@@ -26,6 +26,8 @@ stories.add(
                 <Message type={type}>Hello world!</Message>
                 <br />
                 <MessageLight type={type}>Hello world!</MessageLight>
+                <br />
+                <Notification type={type}>Hello world!</Notification>
```

Comment:
Přemýšlím jestli není lepší to pojmenovat jako `MessageNotification`. Pokud ne, tak bych to poté vytvořil jako obecnou komponentu a ne jako view Message komponenty.